rslib/backend.proto

```diff
@@ -1052,6 +1054,19 @@ message FindAndReplaceIn {
   string field_name = 6;
 }
 
+message BrowserColumns {
+  repeated BrowserColumn columns = 1;
+}
+
+message BrowserColumn {
+  string key = 1;
+  string label = 2;
+  bool is_sortable = 3;
+  bool sorts_reversed = 4;
```

Comment:
Not sure if it's a good idea or not, but one other possibility would be to merge the two into an enum of { NO, NORMAL, REVERSED }